about.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,700;1,300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
+    <title>About Me Page</title>
+</head>
+<body tabindex="0">
+    <div class="quick-info">
+        <ul>
+            <li><a href="https://www.facebook.com/profile.php?id=100010729383548"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+            <li><a href="https://twitter.com/tiredashell0"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+            <li>English</li>
+            <li><a href="#home">My Page</a></li>
+            <li>Logout</li>
+        </ul>
+    </div>
```

Comment:
- [ ] It looks like you can improve the way you made the navbar and icons for it. Please work on it. 🐶 
![image](https://user-images.githubusercontent.com/20563252/133697327-d02132c7-82da-415a-93ee-bda52a8afa73.png)

Reply:
please give me more details about this Cause I am kind confused